test/admin/features/question_time_limits.feature

```diff
@@ -8,13 +8,13 @@ Feature: Question time limit tests
   Background:
     Given I am on the admin page
 
-  Scenario: Question time limit has a default value of 5 seconds
+  Scenario: Question time limit has a default value of 6 seconds
```

Comment:
i did wonder if this would break due to the recent changes.  
Could you make this dynamic and read it from the db upfront? 🤔

Reply:
Good idea, ill put that in now👌

Reply:
At one point we weren't so sure that there would be a setting in the DB at all, so there is a config fallback, and possibly a fallback if the config is missing.

I'd say it should just be in the DB, and we fail to start if it's missing.  This is a bit tangential to this work, so if we agree we can raise another PR for it.

Reply:
ah yes, of course.  So basically its embedded within the admin app.

Agree with what you suggest, and ill make a tech debt item for it.

Reply:
Happy to do this as a separate piece of work...if guy agrees

Reply:
yes, probably best.

Reply:
Sweet awaiting reciew from amit and a green build